src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -546,6 +546,105 @@ def another_user(user=None, tenant=None, account=None):
                       calling_format='boto.s3.connection.OrdinaryCallingFormat')
     return conn, arn
 
+
+def list_topics_by_admin(assert_len, tenant=''):
```

Comment:
i think you can remove the `_by_admin` suffix of the functions names.
also, i think you can make the "assert" parameters optional. so these functions could be used in cases where the caller does not know what to expect. e.g.
```python
def list_topics_by_admin(assert_len=None, tenant=''):
    if tenant == '':
        result = admin(['topic', 'list'], get_config_cluster())
    else:
        result = admin(['topic', 'list', '--tenant', tenant], get_config_cluster())
    parsed_result = json.loads(result[0])
    if assert_len:
        assert_equal(len(parsed_result['topics']), assert_len)
    return parsed_result
```